File: gaia-sdk-javascript/src/graphql/request/input/CompleteBinaryWriteImpulse.ts
export class CompleteBinaryWriteImpulse {
    private uri: string;
    private uploadId: string;
    private chunks: Array<string>;

    constructor(uri: string, uploadId: string, chunks: Array<string>) {
        this.uri = uri;
        this.uploadId = uploadId;
        this.chunks = chunks;
    }
}

File: gaia-sdk-java/gaia-sdk-api/src/main/kotlin/gaia/sdk/api/data/request/CompleteBinaryWriteImpulse.kt
package gaia.sdk.api.data.request

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty

data class CompleteBinaryWriteImpulse @JsonCreator constructor(
        @JsonProperty("uri") val uri: String,
        @JsonProperty("uploadId")val uploadId: String,
        @JsonProperty("chunks") val chunks: List<String>)
